src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -362,6 +362,7 @@
 TF_MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING_NAMES = OrderedDict(
     [
         ("layoutlm", "TFLayoutLMForQuestionAnswering"),
+        ("layoutlmv3", "TFLayoutLMv3ForQuestionAnswering"),
```

Comment:
The torch part has this model in `MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING_NAMES`